package/MDAnalysis/analysis/base.py

```diff
@@ -27,6 +27,8 @@
 
 import numpy as np
 import logging
+from MDAnalysis.lib.log import ProgressMeter
```

Comment:
You should install pep8 linter packages then. I can also recommend auto formatters very much.